sessionctx/stmtctx/stmtctx.go

```diff
@@ -197,12 +197,13 @@ type StatementContext struct {
 	EnableOptimizeTrace bool
 	// LogicalOptimizeTrace indicates the trace for optimize
 	LogicalOptimizeTrace *tracing.LogicalOptimizeTracer
-	// WaitLockLeaseTime is the duration of cached table read lease expiration time.
-	WaitLockLeaseTime time.Duration
 	// EnableOptimizerCETrace indicate if cardinality estimation internal process needs to be traced.
 	// CE Trace is currently a submodule of the optimizer trace and is controlled by a separated option.
 	EnableOptimizerCETrace bool
 	OptimizerCETrace       []*tracing.CETraceRecord
+
+	// WaitLockLeaseTime is the duration of cached table read lease expiration time.
+	WaitLockLeaseTime time.Duration
```

Comment:
Why move it?

Reply:
Put the optimizer trace-related fields together.